docs/en/concepts-and-designs/kubesphere-logging.md

```diff
@@ -0,0 +1,46 @@
+# KubeSphere Logging
+
+## Overview
+
+The KubeSphere logging system comprises many components that work together to achieve centralized log management. Once you enable the logging module during installation, a few components will be deployed or created within the namespace `kubesphere-logging-system`:
+
+- Fluent Bit: log collector deployed on each node.
+- Fluent-bit operator: automates tasks for operating Fluent bit instances.
+- Logsidecar injector: injects sidecar into pods for streaming on-disk files to stdout.
+- Elasticsearch cluster: log storage backend.
+- Curator: performs stale log cleanup.
+
+The diagram below presents the overall logging architecture:
+
+![](../../images/kubesphere-logging-overview.png)
+
+The following sections will detail the design and functionality of each component. 
+
+## Log Collection and Forward
+
+There are two types of logs produced by containers:
+
+- stdout and stderr: written to `.log` files in the host directory `/var/log/containers` by default.
+- application logs: files on pod volume, file locations may vary. 
+
+Fluent Bit serves as the node logging agent. It has access to container log directory and works for applications' stdout and stderr. In KubeSphere, we forked and modified the origin Fluent Bit project fluent/fluent-bit. The difference between fluent/fluent-bit and [kubesphere/fluent-bit](https://github.com/kubesphere/fluent-bit) lies that we add a reload interface to support dynamic configuration (see issue [#365](https://github.com/fluent/fluent-bit/issues/365)).
```

Comment:
node logging agent => logging agent?
"It has access to container log directory and works for applications' stdout and stderr." => "It can access host's container log directory and collect container's logs sent to stdout/stderr"